package.json

```diff
@@ -141,6 +141,6 @@
     "start": "node server/server.js",
     "clean": "git clean -f -x -e node_modules/",
     "test": "jest",
-    "lint": "eslint --fix '**/**/*.{ts,tsx}'"
+    "lint": "eslint --fix **/*.{ts,tsx}"
```

Comment:
This does break on my machine, not sure why:
```
PS C:\Users\cynic\Documents\src\improved-initiative> npm run lint

> improved-initiative@2.12.0 lint C:\Users\cynic\Documents\src\improved-initiative
> eslint --fix **/*.{ts,tsx}

The system cannot find the path specified.
```

Reply:
Maybe a Mac vs PC thing? Do you use a Mac?

Reply:
I'm on Windows 10 , generally using Powershell integrated with VS Code.

Reply:
I think I might just leave this for now and switch it back at the end after I've finished making commits. It's really wired that it works for you one way and me another way.

Reply:
Actually I'm going to leave this in on my end because it's going to be annoying to keep switching if I need to make more changes.

When you merge the pull request, you can add the commas back in.